source/_integrations/sun.markdown

```diff
@@ -51,23 +51,26 @@ sun:
 <img src='/images/screenshots/more-info-dialog-sun.png' />
 </p>
 
-## Implementation Details
+## Automation trigger
 
 The sun's event listener will call the service when the sun rises or sets with
 an offset.
 
-The sun event need to have the type 'sun', which service to call,
-which event (sunset or sunrise) and the offset.
+The sun trigger need to have the type 'sun', which event (sunset or sunrise) and an optional offset.
 
 ```json
 {
-    "type": "sun",
-    "service": "switch.turn_on",
+    "platform": "sun",
```

Comment:
This is really oldschool actually. Let's convert it to an actual YAML trigger to make it easier to understand.